phpBB/includes/functions_compatibility.php

```diff
@@ -195,3 +195,15 @@ function get_user_rank($user_rank, $user_posts, &$rank_title, &$rank_img, &$rank
 	$rank_img = $rank_data['img'];
 	$rank_img_src = $rank_data['img_src'];
 }
+
+/**
+ * Get tables of a database
+ *
+ * @deprecated 3.1.0 (To be removed: 3.3.0)
```

Comment:
The function was already deprecated in 3.1.0. I just added the deprecated message and moved the function in `functions_compatibility.php`.